scripts/clean-local-repos.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+repos_dir="${HOME}/repos"
+exclude_dir=("$@")
+
+folders=$(find "${repos_dir}" -maxdepth 1 -mindepth 1 -type d -exec basename {} \;)
```

Comment:
This find command can return folders which are not git repos. You can protect against it, all git repos will have a `.git` folder at their root.